ai-service/main.py

```
@@ -0,0 +1,17 @@
+code = """
+hello world
+"""
+
+try:
+    embedding = embed_text(code)
+    add_chunks([code], [embedding])
+
+    # List all stored items
+    print("Stored documents in DB:")
+    print(collection.peek())
+except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)